src/app/home/home-router.module.ts

```diff
@@ -18,7 +18,8 @@ const routes: Routes = [
       { path: 'community', component: CommunityComponent },
       { path: 'community/:nation', component: CommunityComponent },
       { path: 'resources', loadChildren: '../resources/resources.module#ResourcesModule' },
-      { path: 'meetups', loadChildren: '../meetups/meetups.module#MeetupsModule' }
+      { path: 'meetups', loadChildren: '../meetups/meetups.module#MeetupsModule' },
+      { path: 'notification/view', component: NotificationViewListComponent }

```

Comment:
Similar to the above, the path can simply be 'notifications' if it's the only notification component.